examples/icon-color.js

```diff
@@ -25,7 +25,7 @@ rome.setStyle(new Style({
   image: new Icon({
     color: '#8959A8',
     crossOrigin: 'anonymous',
-    src: 'data/dot.png'
```

Comment:
Curious why we would change this for Rome only.

Reply:
It's only to have a mix of different image type